scss/main.scss

```diff
@@ -31,38 +31,17 @@ p {
 }
 
 .btn-link {
-  cursor: pointer;
-  padding: 0;
+  &.inline-link {
```

Comment:
While I think this could be a hyperlink, this is fine, let's get it merged in. We've got bigger things to deal with.